tools/check-rsyslog-version

```diff
@@ -15,14 +15,16 @@ else
   MINIMAL_RSYSLOG_VERSION="5.7.1"
 fi
 
+CANONIFIED_VERSION=$(echo ${MINIMAL_RSYSLOG_VERSION} | sed -r 's/\./_/g')
+
```

Comment:
comment at begining of module should be updated to tell it can take a parameter.
Plus, does this command work everywhere where we have rsyslog (aix?)